Dockerfile

```diff
@@ -12,16 +21,33 @@ ARG PROTOBUF_VERSION=4.24.4
 
 ARG BASE_IMAGE=nvcr.io/nvidia/pytorch:24.03-py3
 
-FROM ${BASE_IMAGE}
-
-ARG MAX_JOBS
+FROM ${BASE_IMAGE} AS aligner-bump
```

Comment:
if the user starts from a 24.07 image, wouldn't it still reinstall everything else other than aligner?